@uportal/user-profile-menu/src/components/ProfileMenu.vue

```diff
@@ -147,6 +147,8 @@ export default {
 
       img {
         object-fit: cover;
+        height: 100%;
+        width: 100%;
```

Comment:
will setting both `height` and `width` cause image skew issues?
Or does `object-fit` take care of that?

Reply:
object-fit takes care of that